src/routes/events.router.js

```diff
@@ -0,0 +1,13 @@
+/*
+/api/events routes; This will handle all event related api calls.
+*/
+
+module.exports = (app) => {
+    const events = require('../controllers/events.controller.js');
+
+    // create a new event
+    app.post('/api/events', events.validate('create'), events.create);
```

Comment:
the path should be `/api/event` ? `event` should be singular cause it will only create 1 event at a time
or `/api/events/create`